### ISSUE TITLE
Update Account Error

### ISSUE THREAD

**Issue by onielmartinjr** [onielmartinjr/cen4010#40]
2 Issues on Update account page:

-the receive notifications does not change when updating
-there are instances when the save user functionality should work, but it doesn't

**Comment by nickJones67** [comment]
I tested the "receive notifications" problem and it seems to be working ok. When does the save function not work?

**Comment by onielmartinjr** [comment]
@nickJones67 

This is fixed now. Took a hell of a lot of time, but it's done.

So there was no issue with the PHP, this was a database issue where the PHP was passing '1' for a BIT field and the bit equivalent of '{anything}' is always going to be 1, including '0'. I changed the database field to TINYINT and this resolved the issue.

On that note, I updated the database export code to reflect this. I recommend dropping what you have in the DB and applying the new script I have attached. Unfortunately, that means you'll lose your accounts so you'll have to recreate them.

**Comment by onielmartinjr** [comment]
@nickJones67 

ALSO - I learned something new. MySQL will not let you UPDATE a record if you're not changing any of the fields. SO: my assumption was wrong in thinking that if someone goes to update their account, doesn't change anything and hits submit, it would work.

I changed the error messages slightly to reflect this so it doesn't say "at this time" and it just says: your account was not created.
